pages/common/sqsc.md

```diff
@@ -0,0 +1,31 @@
+# sqsc
+
+> A command line AWS Simple Queue Service client.
+
+- List all queues:
+
+`sqsc lq [queue-prefix]`
```

Comment:
I think @agnivade was trying to also ask you to change out the `[` and `]` for `{{` and `}}`.